tests/components/template/conftest.py

```diff
@@ -1,2 +1,27 @@
 """template conftest."""
-from tests.components.light.conftest import mock_light_profiles  # noqa: F401
+import pytest
+
+from homeassistant.setup import async_setup_component
+
+from tests.common import assert_setup_component, async_mock_service
+
+
+@pytest.fixture
+def calls(hass):
+    """Track calls to a mock service."""
+    return async_mock_service(hass, "test", "automation")
+
+
+@pytest.fixture
+async def start_ha(hass, do_count, do_domain, do_config, caplog):
```

Comment:
Why do we prefix with `do_`? What does it mean? Variable names should preferably start with nouns and function names start with verbs.

Reply:
I normally do it to separate parameters from variables, but no big deal.

do_ is gone!